.github/workflows/android-release.yml

```diff
@@ -1,13 +1,17 @@
-name: Android - Demo App Tagged Release
+name: Liquid Auth - Android - Release
 
 on:
   push:
-    tags:
-      - 'v*'
+    # TODO: rename trunk to main
```

Comment:
Is there a reason why we can't do the renaming now?

Reply:
Mainly due to the work that is in progress, didn't want this PR to bleed over into the others.